gtfs/models/route.py

```diff
@@ -1,8 +1,18 @@
 from django.contrib.gis.db import models
+from django.db.models import Subquery
 from django.utils.translation import gettext_lazy as _
 
+from maas.models import MaasOperator
+
 from .agency import Agency
 from .base import GTFSModelWithSourceID
+from .feed import Feed
+
+
+class RouteQueryset(models.QuerySet):
+    def for_maas_operator(self, maas_operator: MaasOperator):
+        feeds = Feed.objects.for_maas_operator(maas_operator)
+        return self.filter(feed__in=Subquery(feeds.values("pk")))
```

Comment:
I'm 88% certain `self.filter(feed__in=feeds)` would produce exactly the same db query.

Reply:
You are probably correct. Looking at this now, I'm not sure what I was thinking. :sweat_smile: